extensions/amp-render/1.0/storybook/Basic.amp.js

```diff
@@ -77,3 +77,32 @@ export const WithRemoteSrc = () => {
 WithRemoteSrc.story = {
   name: 'With remote src',
 };
+
+export const WithBindableSrc = () => {
+  const srcUrl = text(
+    'src',
+    'http://localhost:9001/examples/amp-render-data.json'
+  );
+
+  return (
+    <>
+      <amp-render
+        data-amp-bind-src="bindSrc"
+        src={srcUrl}
+        width="300"
+        height="100"
+        layout="fixed"
+      >
+        <template type="amp-mustache">{`Hi {{name}}!`}</template>
+      </amp-render>
+
+      <button on="tap:AMP.setState({ bindSrc: 'http://localhost:9001/examples/amp-render-data2.json' })">
```

Comment:
@jridgewell @caroqliu @samouri I get the following error when this button is clicked: `Error: XHR Failed fetching (http://localhost:8000/...): Failed to fetch​​​` Not sure why its trying to fetch on port 8000. Any ideas how to resolve this?

Reply:
I don't know either. I'll check out this branch and trace via devtools to figure out where the `8000` is coming from

Reply:
The failure actually isn't coming from `fetchJson`. It is requesting `dist/ww.js` which is the worker required for amp-bind to function properly.

I don't know anything about how Storybooks works...is this story able to declare that it depends on amp-bind?

Reply:
Ah! So its a CORs issue. Fix incoming - you dont need to block on this.